### PR TITLE
Various changes to fix build of containment rpm of SLE12 image

### PR DESCRIPTION
**Pull request by vuntz**

### CODE REVIEW

**Comment by vuntz** [comment]
For some reason, the fdupes bit doesn't work. Not sure why.

File: image.spec.in [hunk 1]
```diff
@@ -29,6 +30,7 @@ chmod -R u+w *
 %install
 install -d -m 755 $RPM_BUILD_ROOT%containment_image_dest_dir
 cp -pr * $RPM_BUILD_ROOT%containment_image_dest_dir
+fdupes $RPM_BUILD_ROOT%containment_image_dest_dir
```
Comment (AbelNavarro) [inline image.spec.in]: this command without options does not take any action on duplicated files. At least that's what I could observe from manual testing. Are we sure this command is needed here?

File: image.spec.in [hunk 2]
```diff
@@ -1,36 +1,39 @@
...
-
+bsdtar xf %{SOURCE0}
+chmod -R u+w *
 
 %install
-install -d -m 755 $RPM_BUILD_ROOT%containment_image_dest_dir
-cp -pr * $RPM_BUILD_ROOT%containment_image_dest_dir
+install -d -m 755 %{buildroot}%{containment_image_dest_dir}
+cp -pr * %{buildroot}%{containment_image_dest_dir}
+fdupes %{buildroot}%{containment_image_dest_dir}
```
Comment (AbelNavarro) [inline image.spec.in]: Just noticed I commented on an outdated commit. It still applies here so let me paste it:

> this command without options does not take any action on duplicated files. At least that's what I could observe from manual testing. Are we sure this command is needed here?

Comment (vuntz) [inline image.spec.in]: Doh. Should be %fdupes. That explains why it was not working as intended :-)

**Comment by aspiers** [comment]
+1

**Comment by rsalevsky** [comment]
+1

**Pull request merged**